packages/utilities/src/hooks/useViewLoading.ts

```diff
@@ -1,16 +1,34 @@
 import { useEffect, useState } from 'react';
 
-export default function useViewLoading(view: __esri.MapView | null) {
+let timeoutId: NodeJS.Timeout | null = null;
```

Comment:
Was there a reason why you didn't use a ref here? I would think that using a module-level variable like this means that it would be shared among all uses of the hook whereas a ref would be scoped to each component that uses the hook. I'm not sure which is better. I'm just curious what your thought process was.

Reply:
I didn't think about that. good catch.